inlang/source-code/paraglide/paraglide-js/src/compiler/compile.ts

```diff
@@ -25,8 +28,14 @@ export const compile = (args: {
 		compileMessage(message, args.settings.languageTags, args.settings.sourceLanguageTag)
 	)
 
-	telemetry.capture({
-		event: "PARAGLIDE-JS compile executed",
+	getPackageJson(fs, process.cwd()).then((packageJson) => {
+		const stack = getStackInfo(packageJson)
+		telemetry.capture({
+			event: "PARAGLIDE-JS compile executed",
+			properties: {
+				stack,
+			},
+		})
 	})
```

Comment:
How is ensured that the event will be captured? 

Shouldn't we change the compile function to be async?